fl4health/clients/basic_client.py

```diff
@@ -382,23 +388,26 @@ def _handle_logging(
             current_epoch (Optional[int]): The current epoch of local training.
             logging_mode (LoggingMode): The logging mode (Training, Validation, or Testing).
         """
-        log(INFO, "")  # An empty log line for aesthetics
 
         initial_log_str = f"Current FL Round: {str(current_round)}\t" if current_round is not None else ""
         initial_log_str += f"Current Epoch: {str(current_epoch)}" if current_epoch is not None else ""
 
         # Maybe add client specific info to initial log string
-        client_str, client_logs = self.get_client_specific_logs()
-        initial_log_str += client_str
+        client_str, client_logs = self.get_client_specific_logs(current_round, current_epoch, logging_mode)
 
         if initial_log_str != "":
```

Comment:
This is perhaps a silly question that I missed in reviewing before, but it seems like there isn't a setting in which this log string is empty (""), since we set it on line 352? If that's correct, maybe we just ditch this if?

Reply:
Ah I see...if both current_round and current_epoch are None then this could be empty.